### DISCUSSION TITLE
"Important.jpg"

### DISCUSSION THREAD

**Discussion by ChargableXwater** [luckydog7/Funkin-android#1397] [General]
Do I even need to say anything I looked the files cuz why not😏 but I found "important.jpg" and just why 👁👄👁

**Comment by Vicdragon** [comment]
"Impotant.jpg" it's the luckydog real face 🤡👌

> Do I even need to say anything I looked the files cuz why not😏 but I found "important.jpg" and just why 👁👄👁

**Comment by luckydog7** [comment]
why not





sus

**Reply by DzGithub2805** [reply]
Funni image

**Reply by CalibriLight** [reply]
Luckydog sus lmao

**Reply by LightningCreeper381** [reply]
sussybaka

**Comment by HighFlyer-222** [comment]
amogus
